core/domain/feedback_jobs_continuous.py

```diff
@@ -179,10 +201,27 @@ def entity_classes_to_map_over(cls):
 
     @staticmethod
     def map(item):
+        """Map function.
+
+        Args:
+            item: FeedbackThreadModel. A feedback thread model instance.
+
+        Yields:
+            A tuple of two elements:
+              - item.exploration_id: str. The exploration id of the item.
```

Comment:
Drop "item.exploration_id: " and "item.status: ", since those are not actually part of the yielded tuple. So:
```
    - str. The exploration id associated to the feedback thread.
    - str. The feedback thread's status.
```